Hazel/src/Hazel/Core/Window.h

```diff
@@ -42,6 +42,8 @@ namespace Hazel {
 		virtual void* GetNativeWindow() const = 0;
 
 		static Scope<Window> Create(const WindowProps& props = WindowProps());
+	public:
+		static float highDPIScaleFactor;
```

Comment:
To keep things in line with naming conventions; this should be ``s_HighDPIScaleFactor``.

Reply:
Yeah that is true. i will change that.